client/payment-methods/index.js

```diff
@@ -125,6 +125,8 @@ const PaymentMethods = () => {
 		.filter( ( id ) => 'link' !== id )
 		.map( ( methodId ) => methodsConfiguration[ methodId ] );
 
+	const isCreditCardEnabled = enabledMethodIds.includes( 'card' );
```

Comment:
@zmaglica the code works as advertised, but I'd like some context included about this line:

1. Is `enabledMethodIds` UPE-specific?
2. Is it the same with split-UPE?
3. Does `enabledPaymentMethodIds` represent available ones, or does UPE force-enable it here?

Reply:
I have tested both with UPE and Split-UPE replicating the steps above and the error was not presented anymore. I think that the `enabledPaymentMethodIds` represents the ones that are checked or selected. The available ones can be found in `availableMethods` or `useGetAvailablePaymentMethodIds()`